nntrainer/compiler/tflite_interpreter.cpp

```diff
@@ -212,20 +223,32 @@ class TfOpIdxMap {
 
   const std::vector<int> &getOutputs() const { return outputs; }
 
+  const std::vector<const Tensor *> &getTensors() const { return tensors; }
+
+  std::ptrdiff_t getTensorIndex(const Tensor *tensor) const {
+    auto it = std::find(tensors.begin(), tensors.end(), tensor);
+    std::ptrdiff_t index = -1;
+    if (it != tensors.cend()) {
+      index = std::distance(tensors.begin(), it);
+    }
+    assert(index != -1);
```

Comment:
```suggestion
    auto it = std::find(tensors.begin(), tensors.end(), tensor);
    assert(it == tensors.end());
    return std::distance(tensors.begin(), it);
```
But, it it right with `assert` ?

If you intended to return -1 in `not found` case,
```
if (it == tensors.end()) return -1;

return std::distance(tensors.begin(), it);
```

Reply:
Oh, -1 case is the case that shouldn't be occurred. So, I'm going to change it to throwing error. Thank you for your review:)